src/Components/Patient/PatientRegister.tsx

```diff
@@ -595,6 +599,8 @@ export const PatientRegister = (props: PatientRegisterProps) => {
         srf_id: state.form.srf_id,
         covin_id: state.form.covin_id,
         is_vaccinated: state.form.is_vaccinated,
+        number_of_doses: Number(state.form.number_of_doses),
+        vaccine_name: state.form.vaccine_name,
```

Comment:
add a check to see if the value is _Select_, `vaccine_name: state.form.vaccine_name && state.form.vaccine_name !== 'Select' ? state.form.vaccine_name : undefined,`

Reply:
@khavinshankar  Thanks. I have made changes.